nekoyume/Assets/_Scripts/UI/Widget/Screen/DimAndMessageLoadingScreen.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Nekoyume.EnumType;
+using Nekoyume.L10n;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI
+{
+    public class DimAndMessageLoadingScreen : ScreenWidget
```

Comment:
How about `DimmedLoadingScreen`?

Reply:
I think that's a better name too. I'll change it!